bigbluebutton-html5/imports/ui/components/profile-settings/component.tsx

```diff
@@ -63,10 +62,6 @@ const intlMessages: { [key: string]: { id: string; description?: string } } = de
     id: 'app.videoPreview.webcamSettingsTitle',
     description: 'Title for the video preview modal',
   },
-  closeLabel: {
```

Comment:
Client is crashing without this one... It may have been removed by mistake

Reply:
This actually depends on this other PR https://github.com/bigbluebutton/bigbluebutton/pull/22127

(I'll fix the conflict on this other one and mark you as reviewer)